roxctl/deployment/check/check.go

```diff
@@ -172,6 +174,12 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
+	if d.cluster != "" && d.namespace == "default" {
```

Comment:
I still think it's valid to extract `"default"` to a const.

Reply:
Done